src/Console/Commands/CountUniqueUser.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Biigle\Modules\Kpis\Console\Commands;
+
+use Biigle\User;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class CountUniqueUser extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'kpis:count-unique-user';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Counts unique user of last month';
+
+    /**
+     * Execute the command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $nbrUser = $this->countUniqueUser();
+
+        $date = Carbon::now()->subMonth()->endOfMonth()->toDateString();
+
+        DB::transaction(function () use ($nbrUser, $date) {
+            DB::table('kpis_unique_users')->insert(['date' => $date, 'value' => $nbrUser]);
+        });
+    }
+
+    private function countUniqueUser()
+    {
+        $fromDate = Carbon::now()->subMonth()->startOfMonth()->toDateString();
+        $tillDate = Carbon::now()->subMonth()->endOfMonth()->toDateString();
+        return User::whereBetween('login_at', [$fromDate, $tillDate])->get()->count();
```

Comment:
You can use this as a more efficient query:
```suggestion
        return User::whereBetween('login_at', [$fromDate, $tillDate])->count();
```
This counts the rows directly in the database which is much faster.